chapters/appendix-VIII-SPDX-Lite.md

```diff
@@ -0,0 +1,52 @@
+# Appendix-VIII SPDX Lite
+
+## 1. Explanation of SPDX Lite 
+
+The SPDX Lite defines the subset of the SPDX specification, from the point of view of use cases in industries. The SPDX Lite aims at the balance between the SPDX standard and actual workflows in industries. 
+
+The SPDX Lite is the subset of the SPDX specification. The SPDX Lite consists of mandatory part of the Document Creation and Package Information and other basic information. 
+
+The mandatory part of the Package information in the SDPX Lite is basic but indispensable for complying with licenses. It is easy to understand licensing information by reading an SPDX Lite file. It is easy to create manually an SPDX Lite file by anyone who does not have enough knowledge about licensing infomation, so that tools are not necessarily required to create an SPDX Lite file. 
```

Comment:
Please note, here and below there are a few cases where it looks like "SPDX" got the middle letters swapped, so it appears as "SDPX".

Also, I would suggest removing the reference to the SPDX Lite information being "indispensable for complying with licenses." From the perspective of the SPDX project, we are not taking a position on which information is indispensable for license compliance, as every user or distributor of FOSS software may have different perspectives on how to handle compliance. SPDX tries to present a structured way to organize potentially-relevant data while avoiding taking positions on how compliance has to occur.